reviews/elixir_fundamentals_review.livemd

```diff
@@ -147,6 +191,7 @@ defmodule Enumeration do
   [2, 4, 6, 8, 10]
   """
   def double(list) do
+    list |> Enum.map(&(&1 * 2))
```

Comment:
Consider avoiding the pipe operator when there is only one function call. Static analysis tools such as Credo will provide a warning and recommend the following:

```elixir
Enum.map(list, &(&1 * 2))
```